arm-redis/2016-04-01/examples/RedisCacheForceReboot.json

```diff
@@ -12,6 +12,7 @@
   "responses": {
     "200": {
       "body": {
+        "Message": "reboot operation enqueued"
```

Comment:
@TimLovellSmith I understand your concerns and makes sense to wait until next api-version. It's be great if you could please open and issue on that here and from your side as well, we do not miss this on next api-version.